docs/scripts/mdx/utils.ts

```diff
@@ -0,0 +1,76 @@
+import path from "path"
+import { CONSTANT } from "constant"
+import GithubSlugger from "github-slugger"
+import type { DocumentContent, DocumentLevel } from "mdx"
+import type { Document } from "mdx"
+import { otherLocales } from "utils/i18n"
+
+export type EventName = "add" | "addDir" | "change" | "unlink" | "unlinkDir"
+export type DocumentMap = Record<string, Record<string, Document>>
+
+export const MDX_PATH = path.join(process.cwd(), ".mdx")
+export const OTHER_LOCALES = `(${otherLocales.join("|")})`
+
+export function getLocale(filePath: string): string {
+  return (
+    filePath.match(/\.([^\d.]+)\.mdx$/)?.[1] || CONSTANT.I18N.DEFAULT_LOCALE
+  )
+}
+
+export function isDefaultLocale(locale: string) {
+  return locale === CONSTANT.I18N.DEFAULT_LOCALE
+}
+
+export function getSlug(filePath: string): string {
+  return filePath
+    .replace(/^contents\//, "")
+    .replace(new RegExp(`\(/index\)?\(.${OTHER_LOCALES}\)?.mdx$`), "")
+}
```

Comment:
It should replace backslash with `/` on Windows.

```suggestion
export function getSlug(filePath: string): string {
  return filePath
    .replace(/\\/g, "/")
    .replace(/^contents\//, "")
    .replace(new RegExp(`\(/index\)?\(.${OTHER_LOCALES}\)?.mdx$`), "")
}
```